lisa/wlgen/rta.py

```diff
@@ -189,7 +189,7 @@ def _process_calibration(self, calibration):
         return calibration
 
     @classmethod
-    def by_profile(cls, target, name, profile, res_dir=None, default_policy=None,
+    def by_profile(cls, target, profile, name=None, res_dir=None, default_policy=None,
```

Comment:
Should we fix doc/workloads.rst :  "wload = RTA.by_profile(target, "4_50_tasks", profile)"
and also doc/transition_guide.rst: "wload = RTA.by_profile(te, "foo", profile, res_dir, calibration)"
The following example seems outdated as well:
https://github.com/valschneider/lisa/blob/wlgen/pre_pypi_cleanup/lisa/wlgen/rta.py#L227

Reply:
Yes indeed!